templates/tab/js/dashboard/src/internal/addNewScopes.js

```diff
@@ -1,11 +1,11 @@
 import { loginAction } from "./login";
-import FxContextInstance from "./singletonContext";
 import { ErrorWithCode } from "@microsoft/teamsfx";
+import { TeamsUserCredentialContextInstance } from "./singletonContext";
 
 export async function addNewPermissionScope(addscopes) {
-    let teamsfx = FxContextInstance.getTeamsFx();
+    const credential = TeamsUserCredentialContextInstance.getInstance().getCredential();
```

Comment:
remove `.getInstance()` as it is not a valide function of `TeamsUserCredentialContextInstance`
same in ts version

Reply:
updated